src/emulate.c

```diff
@@ -1018,6 +1107,8 @@ static void __trap_handler(riscv_t *rv)
         assert(insn);
 
         rv_decode(ir, insn);
+        reloc_enable_mmu_jalr_addr = rv->PC;
```

Comment:
Is `reloc_enable_mmu_jalr_addr` generally available? It should be SYSTEM specific.

Reply:
Yes, it is. Thanks for pointing out.

Reply:
ICYMI, is the `__trap_handler` already conditionally built by SYSTEM?